android/upload_symbols.gradle

```diff
@@ -0,0 +1,246 @@
+        println('Instabug: INSTABUG_APP_TOKEN found')
+
+        // Get the API KEY if provided
+        if (System.getenv('INSTABUG_API_KEY') == null && properties.getProperty('INSTABUG_API_KEY') == null) {
+            println('Instabug: err: INSTABUG_API_KEY not found. Make sure you\'ve added the environment variable INSTABUG_API_KEY')
+            return null
+        }
+        apiKey = System.getenv('INSTABUG_API_KEY') != null ? System.getenv('INSTABUG_API_KEY') : properties.getProperty('INSTABUG_API_KEY')
+        println 'Instabug: INSTABUG_API_KEY found'
+
+        // Get root host directory path
+        String rootProjectPath = project.properties['rootDir']
+        File rootFile = new File(rootProjectPath)
+
+        // Get the path of the symbols directory
+        String obfuscationPath = project['split-debug-info']
+        File obfuscationFile = new File(obfuscationPath)
+        String symbolsPath
+
+        // If the provided path is relative
+        // append the symbols directory to the root project path to get the full path
+        if (obfuscationFile.absolute) {
+            symbolsPath = obfuscationPath
+        } else {
+            symbolsPath = rootFile.parent + File.separator + obfuscationPath
+        }
+        return [symbolsPath, applicationToken, apiKey]
+    } catch (Exception e) {
+        println("Instabug err: getting config: ${e.message}")
+    }
+}
+
+/**
+    * search for the symbols files in the symbols directory
+    * and create a zip file with the symbols files
+    * @param symbolsPath the path of the symbols directory
+    * @return zipFile the zip file created
+*/
+
+File prepare_zip_file(String symbolsPath) {
+    // Check if the symbols directory provided exists
+    File symbolsDir = new File(symbolsPath)
+    if (!symbolsDir.exists()) {
+        println 'Instabug: err: Symbols directory not found'
+        return null
+    }
+
+    // Search in symbolsPath for files with extension .symbols and add them to a list
+    List<File> symbolsFiles = symbolsDir.listFiles().findAll { file -> file.path.endsWith('.symbols') }
+    if (symbolsFiles.empty) {
+        println 'Instabug: err: No symbols files found'
+        return null
+    }
+    println "Instabug: ${symbolsFiles.size()} Symbols files found"
+
+    // Create a zip file with the symbols files
+    File zipFile = zip_files(symbolsPath, symbolsFiles)
+
+    return zipFile
+}
+/**
+    * create a zip file with the symbols files
+    *
+    * @param symbolsPath the path of the symbols directory
+    * @param symbolsFiles the list of symbols files
+    * @return zipFile the zip file created
+*/
+
+File zip_files(String symbolsPath, List<File> symbolsFiles) {
+    File zipFile = new File(symbolsPath + '.zip')
+    ZipOutputStream zipOutputStream = new ZipOutputStream(new FileOutputStream(zipFile))
+    symbolsFiles.forEach { file ->
```

Comment:
Use `it` instead of `file`.

Reply:
I feel like this isn't really a requirement, more of a preference. Correct me if I'm wrong

Reply:
Yes, both work perfectly fine. 
I didn't express it enough as a suggestion, my bad.